src/journal-remote/journal-upload.c

```diff
@@ -681,6 +756,21 @@ static int parse_argv(int argc, char *argv[]) {
                                 return log_error_errno(r, "Failed to add paths: %m");
                         break;
 
+                case ARG_HEADERS:
+                        r = strv_headers_replace_consume(&arg_headers, optarg);
+                        if (r < 0)
+                                return log_error_errno(
+                                                r,
+                                                "--headers value should contain '='-delimited pairs, separated by comma");
```

Comment:
in our docs and log messages we generally suffix cmdline params that take arguments with `=` to indicate they take a parameter. Hence, in this log msg write `--headers=` rather than `--headers`